tests/foreman/cli/test_satellitesync.py

```diff
@@ -16,95 +16,121 @@
 
 :Upstream: No
 """
-import json
+import os.path
 from random import randint
 
 import pytest
-from fauxfactory import gen_integer
 from fauxfactory import gen_string
-from nailgun import entities
 
 from robottelo import manifests
 from robottelo import ssh
 from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.content_export import ContentExport
+from robottelo.cli.content_import import ContentImport
 from robottelo.cli.contentview import ContentView
 from robottelo.cli.factory import make_content_view
 from robottelo.cli.factory import make_lifecycle_environment
 from robottelo.cli.factory import make_org
 from robottelo.cli.factory import make_product
 from robottelo.cli.factory import make_repository
 from robottelo.cli.package import Package
-from robottelo.cli.puppetmodule import PuppetModule
 from robottelo.cli.repository import Repository
 from robottelo.cli.repository_set import RepositorySet
 from robottelo.cli.settings import Settings
 from robottelo.cli.subscription import Subscription
-from robottelo.config import settings
+from robottelo.constants import CONTAINER_REGISTRY_HUB
 from robottelo.constants import DEFAULT_CV
-from robottelo.constants import ENVIRONMENT
 from robottelo.constants import PRDS
 from robottelo.constants import REPOS
 from robottelo.constants import REPOSET
 
 
 @pytest.fixture(scope='class')
 def class_export_directory():
-    """Create a directory for export, configure permissions and satellite settings,
-    at the end remove the export directory with all exported repository archives"""
-    export_dir = gen_string('alphanumeric')
-
-    # Create a new 'export' directory on the Satellite system
-    result = ssh.command(f'mkdir /mnt/{export_dir}')
-    assert result.return_code == 0
-
-    result = ssh.command(f'chown foreman.foreman /mnt/{export_dir}')
-    assert result.return_code == 0
-
-    result = ssh.command(f'ls -Z /mnt/ | grep {export_dir}')
-    assert result.return_code == 0
-    assert len(result.stdout) >= 1
-    assert 'unconfined_u:object_r:mnt_t:s0' in result.stdout[0]
-
-    # Fix SELinux policy for new directory
-    result = ssh.command(f'semanage fcontext -a -t foreman_var_run_t "/mnt/{export_dir}(/.*)?"')
-    assert result.return_code == 0
-
-    result = ssh.command(f'restorecon -Rv /mnt/{export_dir}')
-    assert result.return_code == 0
-
-    # Assert that we have the correct policy
-    result = ssh.command(f'ls -Z /mnt/ | grep {export_dir}')
-    assert result.return_code == 0
-    assert len(result.stdout) >= 1
-    assert 'unconfined_u:object_r:foreman_var_run_t:s0' in result.stdout[0]
-
-    # Update the 'pulp_export_destination' settings to new directory
-    Settings.set({'name': 'pulp_export_destination', 'value': f'/mnt/{export_dir}'})
-    # Create an organization to reuse in tests
-    yield export_dir
-    ssh.command(f'rm -rf /mnt/{export_dir}')
+    """Grab export directory at the end remove the export directory with
```

Comment:
Maybe I'm just reading the diff wrong, but this function has no content other than the docblock.

Reply:
This fixture is still used by multiple test functions, so I think you've got some missing content here.

Reply:
Must have missed that, I'll delete since that docblock functionality is in another method.